src/I18n/I18n.php

```diff
@@ -201,6 +201,18 @@ public static function config($name, callable $loader)
     }
 
     /**
+     * Registers a custom fallback translator loader
+     *
+     * @param callable $loader A callable object that should return a Package
+     * instance to be used for assembling a new translator.
+     * @return void
+     */
+    public static function fallbackConfig(callable $loader)
```

Comment:
From user's perspective this method does pretty much same thing as `I18n::config()`, just without specifying domain name (e.g. for all domains), so I thought `I18n::config()` and `I18n::fallbackConfig()` would make sense.